cinema/serializers.py

```diff
@@ -31,33 +31,71 @@ class Meta:
 
 
 class MovieSerializer(serializers.ModelSerializer):
+    image = serializers.CharField(source="movie.image", read_only=True)
```

Comment:
You don't need to include image here - this serializer is used for creating new movie without image as per task requirements. You need to include image only on list and detail actions